### ISSUE TITLE
Inclusion of new fields in the PEOPLE register?

### ISSUE THREAD

**Issue by adrik29** [SpeciesFileGroup/taxonworks#2308]
Dear all:
I have a stand alone project called **OmniPaper** in which I provided biographical info for Opiliones authors along with their list of relevant publications and their PDFs.

To replace/merge this with our project in **TaxonWorks** appears to be simple enough, I just have to figure out a field in TW to include (1) biography of the person, (2) the URL for OmniPaper with that person's photo and (3) captions/credits for the photos.
How can I do this?

**Comment by proceps** [comment]
If source has authors parsed as people, you can use Filter Sources task to find all publication of a particular person. For each person, you can create data attributes, and depictions (I do not see depiction option for person in radial annotator, but this should easily be fixed). If depiction is provided, it could be annotated with different levels of attribution. For each person, you can also provide identifiers, etc.

**Comment by adrik29** [comment]
@proceps = thanks, it would be nice to open this possibility to include images of the researchers. I understand that the text of a short biography should be easy to enter in some annotator field, but not the URL of the image (at least I don't know which field to use to this end).

**Comment by mjy** [comment]
I'd suggest the following:
* Ensure the person is in wikidata, for example: https://www.wikidata.org/wiki/Q1043
* Link the wikidata Q number (identifier) to the person in TaxonWorks.
* Provide the biography in Wikidata, there you are doing it for everyone (not sure if possible)
* Provide a picture to Wikidata, there you are doing it for everybody
You will be able to draw down these data from a wikidata API, and match it to queries in TaxonWorks, to merge it into your website, in theory.

@debpaul please ping people who might be able to comment on this?

**Comment by adrik29** [comment]
@mjy -- that's a fine suggestion -- so I can integrate data from different sources and yet everything would be connected through TaxonWorks. I'll make a  few experiments and see if this is feasible...

**Comment by debpaul** [comment]
@mjy I'd suggest ORCID for living folks (and having them get one if they don't have one). IF they are deceased, then Wikidata great but might not exist and would need creating. Pictures go in Wikimedia then link to Wikidata. I don't think you can put a "biography in Wikidata," rather you link to an existing biography. Wikidata links to existing sources (be those websites, papers, books, etc). In essence, it is a gathering of references about a given person, place, thing, concept. The entity must also meet "notability" requirements. @adrik29 please let me know if you have questions. David Shorthouse can advise us. (@dpsspiders on twitter).

It is not hard work to add references to Wikidata or images to Wikimedia. But it is work, and it does take time to learn how to do it.

**Comment by adrik29** [comment]
@debpaul -- I found it tiresome to enter pics in Wikimedia because of the copyright barrier: too many questions and categories to fill...

**Comment by debpaul** [comment]
> @debpaul -- I found it tiresome to enter pics in Wikimedia because of the copyright barrier: too many questions and categories to fill...

Yes @adrik29 I understand. It'd be great to look for ways to get the wikidata community to help (they will!). But we'd need to point them to the lovely photos (and they need to be CC0 I believe, or maybe CC BY will do, I'd need to check). IF there are lots of images, much of this process can be automated...

**Comment by debpaul** [comment]
> 
> 
> @debpaul -- I found it tiresome to enter pics in Wikimedia because of the copyright barrier: too many questions and categories to fill...

In general, we all need easier ways to provide **metadata.** It is tiresome, but it is also **key to discovery**

**Comment by mjy** [comment]
Workflow for metadata in TaxonWorks is fairly evolved:

* Don't load one image at a time, gather the images you want to load
* Use Tasks -> New image to select a set of metadata, remember this set stays across images added, it has to be applied to each set
* Drag drop 20 people images
* Click the pertinent "Apply" to apply the current set of metadata

* Now, navigate to, or create people then
* Use the radial annotator for a given person, the smart-selector will show you recent images, so it's easy to select the pertinent image and assign it to an individual person.  
 

See also https://github.com/SpeciesFileGroup/taxonworks/issues/2321

**Comment by adrik29** [comment]
@debpaul -- Many years ago I started to compile biographies and pictures of harvestmen workers and organized all this material manually as website.
But later I started to see that it could be done more professionally, as part of a database, linked with other kind of data, instead of a standalone site.
I got many photographs from scanning books and papers or by contacting the authors themselves. 
I usually clash with wiki-whatever because of the copyright neurosis.
That's why I would rather point to an author photo on my website instead of entering them in Wikimedia.

**Comment by jlpereira** [comment]
Sorry, pushed to the wrong issue. Reopening
